test/e2etest/test-config.tfvars.tmpl

```diff
@@ -29,7 +29,7 @@ resource_tags = {
 instance_types = ["m5.xlarge"]
 
 # Desired number of nodes that the node group should launch with initially.
-desired_capacity = 3
+desired_capacity = 1
```

Comment:
We did this on purpose to test multiple EKS nodes in e2e test. Back desired capacity to one will result failure in e2e test as the instance type has not enough resources to provide to all products. If you change this to `1` you need to change the instance type that I don't suggest it. I think we better revert this change and leave it as `3`.

Reply:
This is a part of the end to end test. We will start with the smaller capacity and expect the autoscaler to spin up additional resources. The failures we were having were caused by the missing IAM permissions but after fixing those, we have now a passing build:

https://github.com/atlassian-labs/data-center-terraform/actions/runs/1943941571

We have also added an explicit test for the cluster autoscaler that is more defining our expectations:
https://github.com/atlassian-labs/data-center-terraform/pull/182/files#diff-6afb0bd2b96fbd68692fc79175b6bc446cb66721bbe7f07fb693df9d57fc5bdfR10

The e2e build is now running:
https://github.com/atlassian-labs/data-center-terraform/actions/runs/1949842089